packages/core/src/components/Label/Label.types.ts

```diff
@@ -3,6 +3,10 @@ import Text from "../Text/Text";
 
 export type Sizes = "small" | "medium";
 
+export type LabelKind = "fill" | "line";
+
+export type LabelColor = "primary" | "dark" | "negative" | "positive";
+
 export const mapSizesToTextSize: Record<Sizes, TextType> = {
```

Comment:
I don't think we should put this here as it's not a type, plus I don't think we should export it